app/views/docs/command-line-ci.phtml

```diff
@@ -0,0 +1,41 @@
+<p>
+The Appwrite CLI can be used in a non-interactive and headless manner, without saving configuration or sessions. This is especially useful when you want to automate tasks on a CI server.
+</p>
+<h2><a href="/docs/commandline-ci#CIMode" id="CIMode">CI Mode</a></h2>
+
+<p>You can enable CI mode for the Appwrite CLI by setting the <b>project ID, endpoint, and API Key</b>:</p>
+
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="CLI">
+    <pre class="line-numbers"><code class="prism language-bash" data-prism>appwrite client \
+--endpoint https://[HOSTNAME_OR_IP]/v1 \
+--projectId [YOUR_PROJECT_ID] \
+--key YOUR_API_KEY</code></pre>
```

Comment:
This would be prettier and match our other commands if it was indented.

```suggestion
    --endpoint https://[HOSTNAME_OR_IP]/v1 \
    --projectId [YOUR_PROJECT_ID] \
    --key YOUR_API_KEY</code></pre>
```